spec/turn_spec.rb

```diff
@@ -0,0 +1,181 @@
+require 'spec_helper'
+require 'pry'
+
+RSpec.describe 'iteration 2' do
+  # cards
+  let(:card1) { Card.new(:heart, 'Jack', 11) }
+  let(:card2) { Card.new(:heart, '10', 10) }
+  let(:card3) { Card.new(:heart, '9', 9) }
+  let(:card4) { Card.new(:diamond, 'Jack', 11) }
+  let(:card5) { Card.new(:heart, '8', 8) }
+  let(:card6) { Card.new(:diamond, 'Queen', 12) }
+  let(:card7) { Card.new(:heart, '3', 3) }
+  let(:card8) { Card.new(:diamond, '2', 2) }
+
+  context 'Turn :basic' do
+    let(:deck1) { Deck.new([card1, card2, card5, card8]) }
+    let(:deck2) { Deck.new([card3, card4, card6, card7]) }
+    let(:player1) { Player.new('Megan', deck1) }
+    let(:player2) { Player.new('Aurora', deck2) }
+    context '#init' do
+      it 'exists' do
+        turn = Turn.new(player1, player2)
+        expect(turn).to be_a(Turn)
+      end
+    end
+
+    context '#type' do
+      it 'is :basic when card at 0 is not the same' do
+        turn = Turn.new(player1, player2)
+        expect(turn.type).to eq(:basic)
+      end
+    end
+
+    context '#winner' do
+      it 'determines the winner' do
+        turn = Turn.new(player1, player2)
+        expect(turn.type).to eq(:basic)
+        expect(turn.winner.name).to eq('Megan')
+        turn.pile_cards
+        # require 'pry'; binding.pry
+      end
+    end
+
+    context '#pile_cards' do
+      it 'returns the winner of the hand' do
+        turn = Turn.new(player1, player2)
+        hand_winner = turn.winner
+        expect(turn.type).to eq(:basic)
+        expect(turn.winner).to eq(player1)
```

Comment:
We should also `expect` that the `@spoils_of_war` includes the newly piled cards.